src/Congo.WebApi/Controllers/ProductsController.cs

```diff
@@ -28,12 +29,29 @@ public async Task<IEnumerable<Product>> Get()
             return await _mediator.Send(new GetProductListQuery());
         }
 
+
+        [HttpPost]
+        [Route("~/api/{seller}/products")]
+        public ActionResult<Product> Create(ProductInputModel product)
+        {
+            if (!ModelState.IsValid)
+            {
+                return ValidationProblem(ModelState);
+            }
+
+            var productResult = _mediator.Send(new InsertProductCommand
+                (product.Name, product.Description, product.Price, product.ImageUrl)).Result;
+
+            return Ok(productResult);
+        }
+
         [HttpPost("{id}/purchase")]
         public ActionResult<OrderConfirmationResponse> Purchase(Guid id)
         {
             var orderId = Guid.NewGuid();
 
             return new OrderConfirmationResponse { OrderId = orderId };
+
```

Comment:
Was this added on purpose?

Reply:
Most likely by accident while scrolling. I've removed it.